content/en/docs/tasks/administer-cluster/kubeadm/change-package-repository.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Changing The Kubernetes Package Repository
+content_type: task
+weight: 120
+---
+
+<!-- overview -->
+
+This page explains how to switch from one Kubernetes package repository to another
+when upgrading Kubernetes minor releases. Unlike deprecated Google-hosted
+repositories, the Kubernetes package repositories are structured in a way that
+there's a dedicated package repository for each Kubernetes minor version.
+
+For more information about the Kubernetes community-owned package repositories,
+see ["pkgs.k8s.io: Introducing Kubernetes community-owned package repositories"](/2023/08/15/pkgs-k8s-io-introduction/).
+
+## {{% heading "prerequisites" %}}
+
+This document assumes that you're already using the Kubernetes community-owned
+package repositories. If that's not the case, it's strongly recommend to migrate
+to the Kubernetes package repositories as described in
+["pkgs.k8s.io: Introducing Kubernetes community-owned package repositories"](/2023/08/15/pkgs-k8s-io-introduction/).
```

Comment:
We'd better wait for confirmation of that publication date. Also, this is the wrong URL even if that date is right.

Reply:
Blog post link changed to `/blog/2023/08/15/pkgs-k8s-io-introduction/`